src/diffusers/loaders/lora.py

```diff
@@ -94,12 +93,28 @@ def load_lora_weights(
 
         Parameters:
             pretrained_model_name_or_path_or_dict (`str` or `os.PathLike` or `dict`):
-                See [`~loaders.LoraLoaderMixin.lora_state_dict`].
+                A string (model id of a pretrained model hosted on the Hub), a path to a directory containing the model
+                weights saved with [`ModelMixin.save_pretrained`], or a [torch state
```

Comment:
LoRAs are not saved using `save_pretrained()` actually. In `diffusers`, they are usually saved with `save_lora_weights()`. But we will also support loading LoRAs that don't come in the `diffusers` format. So, I would remove this mention to reduce confusion.